src/types/profile.js

```diff
@@ -307,6 +307,19 @@ export type ExtensionTable = {|
   length: number,
 |};
 
+export type VisualMetrics = {|
+  SpeedIndex: number,
```

Comment:
Hi Greg, it seems like the reason for the Pascal Casing here is because the data source is following suit with `visualmetrics.py` [upstream](https://github.com/sitespeedio/browsertime/blob/master/vendor/visualmetrics.py). Changing the visual metrics case style in `mozilla/browsertime` would make it difficult to upstream to `sitespeedio/browsertime`.